lib/ddtrace/sampling/rule.rb

```diff
@@ -0,0 +1,55 @@
+require 'ddtrace/sampling/matcher'
+require 'ddtrace/sampler'
+
+module Datadog
+  module Sampling
+    class Rule
+      attr_reader :matcher, :sampler
+
+      def initialize(matcher, sampler)
+        @matcher = matcher
+        @sampler = sampler
+      end
+
+      # Evaluates if the provided `span` conforms to the `matcher`
+      # and is accepted by the `sampler`.
+      #
+      # If the `matcher` rejects the `span` this method returns `nil`,
+      # to represent that this rule does not apply to the `span`.
+      #
+      # If the `matcher` returns `true` the `sampler` is invoked next.
+      #
+      # If `sampler` accepts the `span`, this method returns an {Array}
+      # with the sampling decision {Boolean} and a sampling rate {Float}.
+      #
+      # If the concept of sampling rate does not apply to the `sampler`
+      # `nil` is returned as the sample rate.
+      #
+      # @param [Span] span
+      # @return [Array<Boolean, Float>] sampling decision and sampling rate
+      # @return [NilClass] if this rule does not apply
+      #   or `nil` if this rule does not apply
```

Comment:
Not sure this `@return` description makes sense

Reply:
Addressed, remove out of place comment line.